internal/e2e/e2e.go

```diff
@@ -202,11 +203,10 @@ func (b *binary) StateFromFile(filename string) (*states.State, error) {
 // Plan is a helper for easily reading a plan file from the working directory.
 func (b *binary) Plan(path string) (*plans.Plan, error) {
 	path = b.Path(path)
-	pr, err := planfile.Open(path)
```

Comment:
Actually, we might think about also running some e2e tests with encryption disabled. But I suppose it's not worth adding to this PR.